chapters/equations.tex

```diff
@@ -244,13 +244,18 @@ \subsubsection{Defining When-Equations by If-Expressions in Equality Equations}\
 The start-values of the introduced \lstinline!Boolean! variables are defined by the taking the start-value of the when-condition, as above where \lstinline!b! is a parameter
 variable.  The start-value of the special functions \lstinline!initial!, \lstinline!terminal!, and \lstinline!sample! is \lstinline!false!.
 
-\subsubsection{Restrictions on Equations within When-Equations}\label{restrictions-on-equations-within-when-equations}
+\subsubsection{Restrictions on When-Equations}\label{restrictions-on-when-equations}
+
+\paragraph*{Restrictions on where a when-equation may occur:}
 
 \begin{itemize}
 \item
   When-equations shall not occur inside initial equations.
 \item
   When-equations cannot be nested.
+\item
+  When-equations can only occur within if-equations and for-equations if
+  the controlling expressions are exclusively parameter expressions.
```

Comment:
(Ready to try some sentence-based line breaking?)

Reply:
Let's consider this resolved for now by https://github.com/modelica/ModelicaSpecification/pull/2718#issuecomment-731154142.